templates/taxi/confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div class="container mt-5">
+        <h1 class="text-center mb-4">{{ title }}</h1>
+        <form action="" method="post">
+            {% csrf_token %}
+            <div class="text-center">
+                <input type="submit" value="Yes, delete" class="btn btn-danger">
```

Comment:
Add a `Cancel` button apart from the confirming one that should lead to the previous page according to the checklist. Update your screenshot accordingly.